scripts/Dockerfile

```diff
@@ -13,16 +18,45 @@ RUN add-apt-repository ppa:rock-core/qt4 -y && \
 RUN rm -rf /catkin_ws && \
     mkdir -p /catkin_ws/src && \
     cd /catkin_ws && \
-    catkin init
+    catkin init && \
+    cd /catkin_ws/src && \
+    git clone https://github.com/RUFFY-369/CrazyS.git && \
+    git clone https://github.com/RUFFY-369/mav_comm.git && \
+    cd /catkin_ws
 COPY .rosinstall /tmp/
 RUN vcs import /catkin_ws/src  < /tmp/.rosinstall
 
+
+#CrazyS
+RUN sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
+RUN apt install -y curl
+RUN curl -s https://raw.githubusercontent.com/ros/rosdistro/master/ros.asc | apt-key add -
+RUN apt update
+RUN apt install -y ros-noetic-joy ros-noetic-octomap-ros ros-noetic-mavlink
+RUN apt install -y ros-noetic-octomap-mapping ros-noetic-control-toolbox
+RUN apt install -y python3-vcstool python3-catkin-tools protobuf-compiler libgoogle-glog-dev
+# RUN rosdep init
+RUN rosdep update
+RUN cd /catkin_ws/src 
+# RUN catkin_init_workspace
+RUN cd /catkin_ws && rosdep install --from-paths src -y -i
+RUN rosdep update
```

Comment:
is this working? Each command is executed from '/' unless you change the working directory (WORKDIR dockerfile command), which I haven't seen

Reply:
yes, it's working and regarding cd command I think WORKDIR isn't needed as full path is mentioned. Also, `rosdep update` is itself done in the root